### ISSUE TITLE
Figure out all remarks to exclude for AR run

### ISSUE THREAD

**Issue by KyleLuoma** [KyleLuoma/f2s_project#23]
NZ is for ARNG, so exclude that one
89 is already excluded

**Comment by KyleLuoma** [comment]
@jessekling You may be able to assist with this one. Would like to know what RMK codes should be excluded when matching AR personnel. Example would be code 89 ARNG AGR position remarks codes.

**Comment by jessekling** [comment]
Shouldn't this be inclusive not exclusive?  All AGRs should have a remark 92 on the position or be put in a templete.  What are your thoughts? @KyleLuoma

**Comment by KyleLuoma** [comment]
@jessekling Probably _should_ be inclusive; but if you look at the most recent ar_f2s analysis file, you'll see that most perfect matches, where APART assigns a Soldier to a UIC, PARA, LN, those positions do not have a 92 remark code. We need to do more analysis on the functional side to discover why people are slotting soldiers in APART to positions without remark codes.

**Comment by jessekling** [comment]
Sounds good, i think that should be our first step, stage one should include the remark code shouldn't it?  and then anything without the code should be stage 2?  what are your thoughts?

**Comment by KyleLuoma** [comment]
Yeah, I think that's a smart approach. I will need to refactor the code a bit. Probably remove the AGR only full_match() function call and replace it by adding a stage 1 AGR match stage and include new match criteria in the matches configuration file. 
So I am thinking the criteria would be AGR89 and AGR92. AGR92 would be true for a position where it has a RMK code = 92. That would be a column in the spaces file. In the faces file, I would create an AGR boolean that is true for all RCC = AGR. So during the stage one match, both AGR92 in spaces and AGR in faces must be true for a match to occur
